protoc-gen-pbandk/jvm/build.gradle.kts

```diff
@@ -30,7 +30,9 @@ tasks.withType<JavaCompile> {
 
 val bootJar by tasks.getting(BootJar::class) {
     archiveClassifier.set("jvm8")
-    launchScript()
+    launchScript {
+		script = file("launch.script")
```

Comment:
Please add a comment with a permalink to the upstream source for this script and a brief (one sentence is fine) explanation of what changes were made to the upstream script.